apps/andi/lib/andi_web/live/dataset_live_view/metadata_form.ex

```diff
@@ -62,11 +62,8 @@ defmodule AndiWeb.EditLiveView.MetadataForm do
       <div class="form-section">
         <%= f = form_for @changeset, "#", [phx_change: :validate, phx_submit: :save, as: :form_data] %>
           <%= hidden_input(f, :orgName) %>
-          <%= hidden_input(f, :orgTitle) %>
```

Comment:
I don't believe the deleted hidden inputs are needed because visible inputs exist for each of these items. I'm assuming these are left over from an initial iteration of the form. I could absolutely be wrong here, so please correct me if I am.

Reply:
undid this and explored a different options for safety